collector/network_route_linux.go

```diff
@@ -202,3 +230,88 @@ func networkRouteProtocolToString(protocol uint8) string {
 	}
 	return "unknown"
 }
+
+func routeTableIDToString() (map[int]string, error) {
+	routeTableName := make(map[int]string)
+	// Paths are optional and do not have to exist
+	rt_tablesConfigFile := "/etc/iproute2/rt_tables"
```

Comment:
Is there a way to retrieve this from proc/sysfs or using ioctls? We want to avoid reading arbitrary files like this.

Reply:
Not that I am aware of. As far as I know, the kernel itself only deals with the route table id. The route table id to name mapping is purely an iproute2 userspace feature to present this in a more user-readable format. 

Someone else please correct me if I am wrong on this.